openedx/core/djangoapps/schedules/message_type.py

```diff
@@ -1,11 +0,0 @@
-import logging
```

Comment:
Won't we still want this in the future to enable logging at the message-level?  (We may want to log only certain types of messages, for example.)
Which is still different from what is happening in this PR - which is enabling all debug-logging for a certain duration.

Reply:
Maybe? I couldn't figure out exactly what that would look like. The use case I know I have is that I would want detailed logging for 1% or 100% of all messages. It would be interesting to think about how we might enable it for just one user or just one course, but I suspect that would require us creating a database model of some kind (since waffle doesn't have access to an HTTP request at this point).